### DISCUSSION TITLE
Unable to run HPOV cmdlet Get-HPOVDriveEnclosureInventory

### DISCUSSION THREAD

**Discussion by Baskervile-Bytes** [HewlettPackard/POSH-HPOneView#582] [Q&A]
Hi,
I am unable to run the following HPOV CMDLET against one of our Synergy frames:
Get-HPOVDriveEnclosureInventory
The command runs and produces the list of drives etc. however completes with the following error and the output objects are unable to be piped to a variable:

Get-HPOVDriveEnclosureInventory : The requested resource '/rest/drive-enclosures/CN773509S8' could not be found. Confirm resource ID/URI and try the call again.

I am using HPOV Module HPOneView.420 as this is unfortunately the current version of the appliance fw

Any help to resolve this would be much appreciated!

**Comment by ChrisLynchHPE** [comment]
OneView 4.20 is well over 2 years old.  We are not making any further patches to that release, including the PowerShell library.  This issue you are running into is caused by stale records within the Index.  Newer libraries have a workaround that addresses this issue.  I would strongly suggest you plan on updating your OneView appliance(s) in order to keep up with not just new features, but bug and security fixes.

**Reply by ChrisLynchHPE** [reply]
Also, this isn't really a discussion, rather a bug report.
